docs/implementation-guide.md

```diff
@@ -162,9 +179,9 @@ To create a repository secret follow this steps.
 ![create-environment-3](./images/create-environment-3.png)
 
 ### Step 9: Run the Deployment GitHub Workflow
-At this point we have completed all of the necessary setup. We are now ready to deploy the PHDI pipelines to your GCP project with Terraform via the provided `Deployment` GitHub Workflow. To run this workflow to deploy the PHDI pipelines to the development environment you created previously, follow the steps below.
+At this point we have completed all of the necessary setup. We are now ready to deploy the PHDI pipelines to your Azure project with Terraform via the provided `Deployment` GitHub Workflow. To run this workflow to deploy the PHDI pipelines to the development environment you created previously, follow the steps below.
 
-1. Navigate to `https://github.com/<MY-GITHUB-ORGANIZATION>/phdi-google-cloud` in your browser.
+1. Navigate to `https://github.com/<MY-GITHUB-ORGANIZATION>/phdi-azure` in your browser.
 2. Click on `Actions` near the center at the top of the page.
 ![navigate-to-actions](./images/navigate-to-actions.png)
 3. Select `Deployment` from the list of Workflows on the left side of the screen.
```

Comment:
Nothing to do here now, but flagging for us to keep in mind that we should add tenth step for running the end to end testing suite once we have it ready.